examples/pytorch/token-classification/run_ner.py

```diff
@@ -202,6 +209,30 @@ def __post_init__(self):
                 assert extension in ["csv", "json"], "`validation_file` should be a csv or a json file."
         self.task_name = self.task_name.lower()
 
+class MyCallback(TrainerCallback):
```

Comment:
How about naming `TBTrainerCallback` instead of `MyCallback` because `MyCallback` does not a specific meaning.

Reply:
Sure, let's me change the callback name.